src/core_plugins/getting_started/public/components/getting_started/getting_started.less

```diff
@@ -0,0 +1,70 @@
+.gettingStartedContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 0;
+}
+
+.gettingStartedOptOutContent {
+  position: absolute;
+  top: 20px;
+  right: 30px;
+}
+
+.gettingStartedWelcome {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 200px;
+  width: 100%;
+  background-color: #f7f7f7;
+
+  h1 {
```

Comment:
🚯 ⛔️ 🚫  Instead of nesting, could you create a new class specific to this element? :)